doc/source/administrator/authentication.md

```diff
@@ -0,0 +1,427 @@
+(authentication)=
+
+# Authentication and authorization
+
+Authentication is about identity, while _authorization_ is about access rights.
+In this section you will learn how to configure both. For example, you can
+configure to authenticate GitHub users and authorize those part of a specific
+GitHub organization.
```

Comment:
Not sure, but I think you're trying to say
"You can configure authentication using GitHub accounts, but restrict what users are authorised to do (or what users can log in?) based on membership of a GitHub organiisation"
?

Reply:
What do you think about 3ea9ded0?